### ISSUE TITLE
Task: Link the API with CMI LDAP

### ISSUE THREAD

**Issue by ReinderVosDeWael** [childmindresearch/ctk-api#4]
### Description

For user authentication, we'll need to hook into CMI's LDAP. No user database need be maintained.

- This task is blocked by https://github.com/orgs/cmi-dair/projects/3/views/3?pane=issue&itemId=38407437.

### Tasks

- [ ] Create a authentication dependency that pulls an auth token from the header.
- [ ] Add the authentication dependency to every endpoint. 
- [ ] Log the user identity on every request. 

### Freeform Notes

This task is currently pre-mature and should remain on hold until the project has progressed further.

**Comment by ReinderVosDeWael** [comment]
See childmindresearch/agate for an example.
